pkg/config/logs/log.go

```diff
@@ -108,10 +109,27 @@ func SetupLogger(loggerName LoggerName, logLevel, logFile, syslogURI string, sys
 	}
 	_ = seelog.ReplaceLogger(loggerInterface)
 	log.SetupLogger(loggerInterface, seelogLogLevel)
-	scrubber.AddStrippedKeys(cfg.GetStringSlice("flare_stripped_keys"))
+	scrubber.AddStrippedKeys(MergeAdditionalKeysToScrubber(
+		cfg.GetStringSlice("flare_stripped_keys"),
+		cfg.GetStringSlice("scrubber.additional_keys")))
```

Comment:
We should print a message when the `flare_stripped_keys` is used to push users to migrate to the new keys.

Reply:
only if there is something in `flare_stripped_keys` then right ?